source/GameData.cpp

```diff
@@ -104,7 +104,8 @@ namespace {
 	bool preventSpriteUpload = false;
 
 	// Tracks the progress of loading the sprites when the game starts.
-	int spriteLoadingProgress = 0;
+	std::atomic<bool> queuedAllImages = false;
+	std::atomic<int> spriteLoadingProgress = 0;
```

Comment:
If this is the number of sprites loaded, that should be what it's called.
```suggestion
	std::atomic<int> loadedSprites = 0;
```

Reply:
Done